challenges/objects-arrays.js

```diff
@@ -88,32 +121,43 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
-const animalNames = [];
-console.log(animalNames);
+let animalNames = zooAnimals.map( zooAnimal => {
+  return zooAnimal.animal_name , zooAnimal.scientific_name
+});
 
-/* Request 2: .map()    
+console.log(animalNames)
 
-The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
+//  Request 2: .map()    
 
-*/
+// The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
+
+
+
+const lowerCase = zooAnimals.map(zooAnimals => {
+  return zooAnimals
+})
```

Comment:
This is the right method here, but we actually aren't manipulating the data. The goal here was to return their names only converted to lowercase.